lua/sim/Prop.lua

```diff
@@ -1,6 +1,11 @@
 local PlayReclaimEndEffects = import('/lua/EffectUtilities.lua').PlayReclaimEndEffects
 
-local minimumLabelMass = 10
+local minimumLabelMass, minimumLabelEnergy = 10, 10
+
+---@class PropSyncData
+---@field mass number
+---@field energy number
+---@field position Vector
```

Comment:
I like the idea of passing energy too

Reply:
It's the result of a Discord discussion--I ended up suggesting we sync prop energy along with mass so that UI mods can use it while we figure out how what we want to deal with it. Was the first time I had a highly-upvoted comment on Discord, which was something I didn't think even made sense for the platform. So here we are.

Reply:
Yes, I don't think we should have labels for energy. But I am thinking of a scanning feature that shows the worth around your mouse (in combination with the range rings of ACS). That way you get an idea about how much reclaim an engineer can attain at a given position.